src/testFixtures/kotlin/uk/gov/justice/digital/hmpps/educationandworkplanapi/DeepCopy.kt

```diff
@@ -0,0 +1,18 @@
+package uk.gov.justice.digital.hmpps.educationandworkplanapi
+
+import com.fasterxml.jackson.databind.DeserializationFeature
+import com.fasterxml.jackson.databind.ObjectMapper
+import com.fasterxml.jackson.databind.SerializationFeature
+import com.fasterxml.jackson.datatype.jsr310.JavaTimeModule
+
+private val OBJECT_MAPPER: ObjectMapper =
+  ObjectMapper().registerModule(JavaTimeModule()).disable(SerializationFeature.WRITE_DATES_AS_TIMESTAMPS)
+    .configure(DeserializationFeature.FAIL_ON_UNKNOWN_PROPERTIES, false)
+
+/**
+ * Extension function to perform a deep copy on any object.
+ * Uses Jackson to serialize then deserialize to create the new object instance.
+ */
+fun <T> T.deepCopy(): T {
```

Comment:
This is a test support function that allows us to create deep copies of objects - a couple of the tests in this PR use this new function